.eslintrc.js

```diff
@@ -10,11 +10,6 @@ module.exports = {
     'plugin:import/errors',
     'plugin:import/typescript',
   ],
-  globals: {
-    Lang: 'readonly',
-    TSMLReactConfig: 'readonly',
-    Translation: 'readonly',
-  },
```

Comment:
i could not get jest to recognize the global types defined in `index.d.ts` so they're no longer global - also `TSMLReactConfig` got renamed to `Settings`